aws-athena/views/reporting-vw_ratio_stats.sql

```diff
@@ -6,8 +6,10 @@ WITH classes AS (
     SELECT
         par.parid,
         par.taxyr,
-        par.class,
-        CASE WHEN par.class IN ('299', '399') THEN 'CONDO'
+        REGEXP_REPLACE(par.class, '[^[:alnum:]]', '') AS class,
```

Comment:
**issue:** Likewise on adding a not null test here.

Reply:
I'll address this [here](https://github.com/ccao-data/data-architecture/pull/367).